test/numbers.jl

```diff
@@ -1159,6 +1159,17 @@ Base.@irrational i46051 4863.185427757 1548big(pi)
     # issue #46051
     @test sprint(show, "text/plain", i46051) == "i46051 = 4863.185427757..."
 end
+
+@testset "Irrational round, float, ceil" begin
+    using .MathConstants
+    @test round(π) == 3.0
```

Comment:
```suggestion
    @test round(π) === 3.0
```

These should be strict `===` comparisons because `3 == 3.0` but `3 !== 3.0`.

Reply:
Have done this in the latest commit